python/foglamp/tasks/north/sending_process.py

```diff
@@ -303,67 +220,60 @@ class SendingProcess:
             "type": "integer",
             "default": "60"
         },
-        "sleepInterval": {
-            "description": "Time in seconds to wait between duration checks",
-            "type": "integer",
-            "default": "1"
-        },
         "source": {
-            "description": "Source of data to be sent on the stream. "
-                           "May be either readings, statistics or audit.",
+            "description": "Source of data to be sent on the stream. May be either readings, statistics or audit.",
             "type": "string",
-            "default": _DATA_SOURCE_READINGS
+            "default": "readings"
```

Comment:
so we can delete  `      _DATA_SOURCE_READINGS = "readings" ` ?

Reply:
what has been done :) _DATA_SOURCE_READINGS is being used as const in this file. either we should remove them all or keep that here too